README.md

```diff
@@ -42,6 +45,7 @@ steps:
       version: ${{ github.sha }}
     env:
       ROLLBAR_ACCESS_TOKEN: ${{ secrets.ROLLBAR_ACCESS_TOKEN }}
+      ROLLBAR_USERNAME: ${{ secrets.ROLLBAR_USERNAME }}
```

Comment:
One question for the shake of curiosity, for your use case do you plan to always use the same username when sending a deploy notification? 
I don't know if you tried, but maybe you could use the `github.actor`, the documentation says:

```
github.actor | string | The login of the user that initiated the workflow run.
``` 

So, in that way, the username send to Rollbar would be dynamic, what do you think?

Reply:
Yeah was thinking the same thing. I guess we can update it would also recommend that the username matches rollbar user.